snippets/base/tests/test_models.py

```diff
@@ -586,6 +587,9 @@ def test_get_admin_url(self):
 
     def test_modified_date_updates_when_template_updates(self):
         snippet = ASRSnippetFactory()
+        # Must refresh from db to get the actual datetime stored in db which
```

Comment:
that sounds like a bug that took a while to find ಠ_ಠ

Reply:
Oh yes, indeed.